projects/arlas-components/src/lib/components/mapgl/draw/modes/strip/strip.direct.mode.ts

```diff
@@ -271,18 +271,20 @@ stripDirectSelectMode.dragRotatePoint = function (state, e, delta) {
     const n = state.rotation.centers.length;
     const cIdx = (this.coordinateIndex(state.selectedCoordPaths) + 1) % n;
     const cCenter = state.rotation.centers[cIdx];
-    const center = point(cCenter);
-    const heading1 = rhumbBearing(center, m1);
+    const cp = point(cCenter);
+    const heading1 = rhumbBearing(cp, m1);
     const heading0 = state.rotation.headings[cIdx];
     const rotateAngle = heading1 - heading0; // in degrees
     const rotatedFeature = transformRotate(state.rotation.feature0,
         rotateAngle,
         {
-            pivot: center,
+            pivot: cp,
             mutate: false,
         });
     state.start = undefined;
     state.feature.incomingCoords(rotatedFeature.geometry.coordinates);
+    const bearingAngle = rhumbBearing(point(rotatedFeature.geometry.coordinates[0][0]), point(rotatedFeature.geometry.coordinates[0][3]));
```

Comment:
We need to check if `rotatedFeature.geometry.coordinates[0]` array has at least 4 elements in order to access the indices 0 and 3, no ?

Reply:
Hello these values are hard coded, so i assume that they ll always be there.

Reply:
ah I didn't see that they were hardcoded. Where is that ?